signer/src/transaction_signer.rs

```diff
@@ -800,7 +800,7 @@ mod tests {
             context,
             context_window: 6,
             num_signers: 7,
-            signing_threshold: 5,
+            signing_threshold: 3,
```

Comment:
Nice, but what is the "real" fix? What are the underlying changes that we need to make for these tests to pass as is? We can make those changes in a separate PR.

For this PR I'd add open a tech debt ticket with a brief description of the issue, and add a TODO comment here (and in the equivalent change in the transaction coordinator), pointing to the ticket number.

Reply:
Done